nullaway/src/main/java/com/uber/nullaway/jarinfer/JarInferStubxProvider.java

```diff
@@ -0,0 +1,19 @@
+package com.uber.nullaway.jarinfer;
+
+import java.util.List;
+
+/** Interface to be implemented by providers of stubx files to be read by NullAway */
+public interface JarInferStubxProvider {
+
+  /**
+   * Returns a list of paths to stubx files to be loaded. Paths must be <b>absolute</b> within the
```

Comment:
Here I chose to require the paths to be absolute, which gives maximum flexibility but is maybe a little confusing / ugly.  An alternative would be to require a path relative to the class implementing the interface.  So if a jar included class `com.foo.bar.MyStubxProvider`, the `.astubx` files would need to be somewhere under `com/foo/bar` and relative paths could be given.  I'm fine either way.

Reply:
> I don't think absolute paths are a problem. We can revisit this later, but wouldn't worry about relative paths right now (in theory, if we ever allow relative paths, paths starting with `/` will still be considered "absolute" relative to the root of the jar, no?)

Ok, sounds good.  Yes, a path starting with `/` would still be treated as "absolute."